packages/pf4-component-mapper/demo/index.html

```diff
@@ -4,8 +4,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" type="text/css" href="https://unpkg.com/@patternfly/patternfly@4.50.4/patternfly-base.css"/>
-  <link rel="stylesheet" type="text/css" href="https://unpkg.com/@patternfly/patternfly@4.50.4/patternfly-addons.css"/>
+  <link rel="stylesheet" type="text/css" href="https://unpkg.com/@patternfly/patternfly@latest/patternfly-base.css"/>
```

Comment:
make sure to update the PF4 docs templates. I don't think we have the latest CSS in the templates.